pkg/coscheduling/coscheduling.go

```diff
@@ -166,6 +168,14 @@ func (cs *Coscheduling) PostFilter(ctx context.Context, state *framework.CycleSt
 			waitingPod.Reject(cs.Name(), "optimistic rejection in PostFilter")
 		}
 	})
+
+	pods, err := cs.frameworkHandler.SharedInformerFactory().Core().V1().Pods().Lister().Pods(pod.Namespace).List(
+		labels.SelectorFromSet(labels.Set{v1alpha1.PodGroupLabel: util.GetPodGroupLabel(pod)}),
+	)
+	if err == nil && len(pods) >= int(pg.Spec.MinMember) {
```

Comment:
can we remove the check here? (given `assigned` < pg.Spec.MinMember per L150)

Reply:
bump.

Reply:
I think we can not  remove the check here.

`assigned < pg.Spec.MinMember` check whether the gang has been satisfied and `len(pods) >= int(pg.Spec.MinMember)` check whether there is enough pods to schedule.

When the first pod come, scheduler may list not all pods. We will reject the first pod in preFilter
https://github.com/kubernetes-sigs/scheduler-plugins/blob/6ef6f63a0826b6045dd955eac599c902e396457f/pkg/coscheduling/core/core.go#L153-L156
But we should not add this pod group to Backoff. If we add this pod group to Backoff, all following pods will be rejected.